local_node.sh

```diff
@@ -15,7 +15,8 @@ HOMEDIR="$HOME/.tmp-evmosd"
 TRACE=""
 
 # feemarket params basefee
-BASEFEE=1000000000
+BASEFEE=0.1
```

Comment:
why this is changed of 1e10 instead of 1e12? Also, maybe we should allow `local_node.sh` to run with both 6 and 18 evm denom decimal? 🤔